include/engine/datafacade/datafacade_base.hpp

```diff
@@ -191,6 +191,8 @@ class BaseDataFacade
     virtual util::guidance::EntryClass GetEntryClass(const EdgeID turn_id) const = 0;
 
     virtual bool IsLeftHandDriving(const NodeID id) const = 0;
+
+    virtual bool IsSegregated(const NodeID) const { return false; }
```

Comment:
why not simply pure virtual?

Reply:
To minimize possible changes, but ok, will do pure.

Reply:
We should aim to have this in both versions of the datafacade, so that we have a consistent state over all versions of osrm that can be run.